include/swift/AST/AutoDiff.h

```diff
@@ -109,12 +111,13 @@ class AutoDiffParameterIndices : public llvm::FoldingSetNode {
   ///
   const llvm::SmallBitVector parameters;
 
-  AutoDiffParameterIndices(llvm::SmallBitVector parameters)
-      : parameters(parameters) {}
-
   static AutoDiffParameterIndices *get(llvm::SmallBitVector parameters,
                                        ASTContext &C);
 
+private:
+  AutoDiffParameterIndices(llvm::SmallBitVector parameters)
```

Comment:
```suggestion
  AutoDiffParameterIndices(const llvm::SmallBitVector &parameters)
```

Reply:
Done in https://github.com/apple/swift/pull/21837/commits/6fe2c6458258a66d31c72730d89aed36f297996d.